sample/home.jsx

```diff
@@ -144,7 +145,14 @@ export default class Home extends Component {
           <label for="rd">Radio Box</label>
         </div>
 
-        <Textfield label="hello" fullwidth={true} required />
+        <Textfield
+          label="hello"
+          fullwidth
+          required
+          ref={tb => {
```

Comment:
Same here, it's interesting for testing but maybe shouldn't be in the sample. Should definitely update the docs though, which will be much easier to enforce once the docs source code is on ```master``` because we can just refuse PRs if they don't update docs 👍